mmdet/integration/nncf/compression_hooks.py

```diff
@@ -15,7 +15,8 @@ def after_train_epoch(self, runner):
 
     def before_run(self, runner):
         if runner.rank == 0:
-            print_statistics(self.compression_ctrl.statistics(), runner.logger)
+            statistics = self.compression_ctrl.statistics()
+            runner.logger.info(statistics.to_str())
```

Comment:
This is aligned with NNCF's internal training samples - the compression statistics are printed by default so that the user can verify if the applied compression is correct.